pom.xml

```diff
@@ -547,4 +547,82 @@
             <version>${maven-version}</version>
         </dependency>
     </dependencies>
+    <profiles>
+        <profile>
+            <id>unshaded</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-shade-plugin</artifactId>
+                        <version>3.6.0</version>
+                        <executions>
+                            <execution>
+                                <phase>package</phase>
+                                <goals>
+                                    <goal>shade</goal>
+                                </goals>
+                                <configuration>
+                                    <transformers>
+                                        <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                                            <manifestEntries>
+                                                <Name>rascal</Name>
+                                                <Main-Class>org.rascalmpl.shell.RascalShell</Main-Class>
+                                                <Specification-Version>${project.version}</Specification-Version>
+                                                <Specification-Vendor>http://www.usethesource.io</Specification-Vendor>
+                                            </manifestEntries>
+                                        </transformer>
+                                        <transformer implementation="org.apache.maven.plugins.shade.resource.AppendingTransformer">
+                                            <resource>org/rascalmpl/uri/resolvers.config</resource>
```

Comment:
I think if we don't shade vallang, we don't need to merge these?

Reply:
That makes total sense.